scripts/validator.py

```diff
@@ -0,0 +1,165 @@
+"""
+this script validates the json files against the schema
+"""
+
+import argparse
+import json
+import logging
+import os
+import pathlib
+
+from jsonschema import Draft202012Validator, validators
+
+local_path = pathlib.Path(__file__).parent.parent.resolve().as_posix()
+
+logger = logging.getLogger(__name__)
+
+
+def get_schemas_path(schema_version):
+    return os.path.join(local_path, "schemas", schema_version)
```

Comment:
I would user `pathlib` instead of `os.path` in general.

```suggestion
    return local_path / "schemas" / schema_version
```